### PR TITLE
replace html tags with bbcodes

### PR DESCRIPTION
**Pull request by lycra**
In order to replace the WordPress HTML Codes with BBCODES of WCF:

More info within the thread:
https://community.woltlab.com/thread/247321-html-tags-in-bbcodes-umwandeln-beim-import-aus-wordpress

### CODE REVIEW

File: files/lib/system/exporter/WordPress3xExporter.class.php [hunk 1]
```diff
@@ -417,7 +417,191 @@ public function exportBlogAttachments($offset, $limit) {
 	}
 	
 	private static function fixMessage($string) {
-		$string = str_replace("\n", "<br />\n", StringUtil::unifyNewlines($string));
+		// we wanna get rid of f*ckin html in articles, so we have to remove a lot of crap
```
Comment (TimWolla) [inline files/lib/system/exporter/WordPress3xExporter.class.php]: Without looking at the actual changes (those are for @BurntimeX): These comments *do not* belong here. They don't belong in the version history either. Please remove them and squash all your commits into a single commit (those bugfix commits without a sane commit message just clutter the version history).

**Comment by lycra** [comment]
Not sure how to delete each commit. So i will create a new PR

**Comment by TimWolla** [comment]
> Not sure how to delete each commit. So i will create a new PR

As far as I know this cannot be done with the GitHub Web UI which you seem to be using.

**Comment by lycra** [comment]
Yeah thats why i would need to delete it and open a new PR. 
"justcode" updated the code again.

**Pull request closed without merging**